admin/app/components/target-profiles/target-profile.hbs

```diff
@@ -22,17 +22,29 @@
       <div class="target-profile-section__container">
         <ul>
           <li><span class="bold">ID : </span>{{@model.id}}</li>
-          <li><span class="bold">Date de création : </span>{{format-date @model.createdAt}}</li>
-          <li><span class="bold">Public : </span>{{this.isPublic}}</li>
-          <li><span class="bold">Parcours Accès Simplifié : </span>{{this.isSimplifiedAccess}}</li>
-          <li><span class="bold">Obsolète : </span>{{this.isOutdated}}</li>
-          <li><span class="bold">Est associé à une campagne : </span>{{this.hasLinkedCampaign}}</li>
           <li><span class="bold">Organisation de référence : </span>
             <LinkTo @route="authenticated.organizations.get" @model={{@model.ownerOrganizationId}}>
               {{@model.ownerOrganizationId}}
             </LinkTo>
           </li>
-          <li><span class="bold">{{t "pages.target-profiles.resettable-checkbox.label"}} : </span>{{this.areKnowledgeElementsResettable}}</li>
+          <li><span class="bold">Date de création : </span>{{format-date @model.createdAt}}</li>
+          <li><span class="bold">Public : </span>{{this.isPublic}}</li>
+          <li><span class="bold">Obsolète : </span>{{this.isOutdated}}</li>
+          <li><span class="bold">Parcours Accès Simplifié : </span>{{this.isSimplifiedAccess}}</li>
+          {{#if this.hasLinkedCampaign}}
+            <li><span class="bold">Est associé à une campagne : </span>Oui</li>
+            {{#if this.hasLinkedAutonomousCourse}}
+              <li><span class="bold">Est associé à un parcours autonome : </span>Oui</li>
+            {{/if}}
+          {{else}}
+            <li><span class="bold">Associé à une campagne ou un parcours autonome : </span>Non</li>
+          {{/if}}
+          <li>
+            <span class="bold">
+              {{t
+                "pages.target-profiles.resettable-checkbox.label"
+              }}&#x20;:&#x20;{{this.areKnowledgeElementsResettable}}</span>
```

Comment:
enlever le `{{this.areKnowledgeElementsResettable}}` du span

Changer en booléen (utiliser un helper ember pour afficher oui ou non ?)